### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by MatthewRGonzalez**
This pull includes recent work and documents my hours. The details here are mostly technical and will most likely be discussed in the next few days with @Zeyuan-Xiong .

### CODE REVIEW

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 1]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+        - I-J119: Adding Specific (Note that hardcopy indicates "Of which proxylin is the component material of chief value" I mentioned this under "extra")
+        - 120: Added Specific (Not that hardcopy indicates " Transparent sheets over 0.003 but not over 0.032 inch thick". Added to "extra")
+        - K118,K121 (Added AV)
+- Multiple discussions in progress (1:46 PM)
+
+#### Comparing PDF and Hardcopy for Inconsistencies:
+Before progressing further, I will compare the GENEVA1 on GitHub and the Hardcopy. In response to this [suggestion.](https://github.com/kbuzard/gradualism/pull/52#discussion_r545213)
+
+- Currently up to paragraph 75. No differences between PDF and hardcopy. 
+- I need to know the specifics of how GENEVA1 was coded. Current understanding is that Torquay was digitized by Ross (available in repo). However, not sure if there is a corresponding digitiztion file for Geneva?
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:196]: NO digitalization file for Geneva. @Zeyuan-Xiong did this manually.

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 2]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+- Continue paragraphs for Victor
+- Create list of "New" Entries
+    - Enter entries in Shared file
+
+### For Victor (Cont...)
+
+- p.1529(a) H2094 Considering Adding new entry for silk (AV of 65). Noted in shared file.
+    - Added some suggested AV's here, but again, will come back
+- p.1530 Added some comments. This section should be discussed 
+- Created a sheet within shared file for new entries
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:289]: To clarify: I'm assuming "new" entries are lines you found in the hard copy that were not present in the PDF. If this is right, please be more explicit here in your notes.

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 3]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+**Submitted Hours Up to This Point** 
+
+- Need to compare certain section between Geneva/Torquay.
+- Check for accuracy
+
+
+## Tuesday, Dec. 22
+
+### Overview
+ - Review past work
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:305]: For context, can you include which paragraphs from which round were covered? Going back now is probably too hard, and if so it's not a big deal. But reading through here, it's easy to get lost as to where you were working. Is this all Geneva?

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 4]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+### Overview
+
+- Meet with Victor
+
+### Summary/Todo:
+
+Completed 2 hours
+
+- Insert lines from "'New' Entries" sheet to "Main Data"
+- Check accuracy entire Geneva1
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:332]: This really means that you finished checking, right?

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 5]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+
+- Meet with Victor
+
+### Summary/Todo:
+
+Completed 2 hours
+
+- Insert lines from "'New' Entries" sheet to "Main Data"
+- Check accuracy entire Geneva1 
+- Compare all of Hardcopy vs PDF (Geneva1)
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:333]: Same as above.

AND: what is the difference between "checking accuracy" and "comparing all of Hardcopy to PDF?"

File: 2020_Winter_RA_Report_Gonzalez.md (hunk 4 above)
Comment (kbuzard) [reply 2020_Winter_RA_Report_Gonzalez.md:332]: From below, I think this might mean that you started...

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 6]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+    - Added Textile machinery AV's
+- p.373: Agricultural hand tool section?
+- p.376 Missing entries .. (new products)
+
+
+### Summary/Todo:
+
+Completed 5 hours
+
+- Continue with accuracy check
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:367]: Again, accuracy check OF WHAT?

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 7]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+- p.1206 added new entry
+- p.1302 added av
+- p.1306 added new entry
+- p.1309 added av and fixed av 
+- p.1405 added new entry
+- p.1407 added rates
+
+### Misc. 
+
+- Geneva1 PDF is missing certain pages. (page 85-86)
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:424]: AAARGH. We need to remember about this and figure out if there's an easy fix. We can't keep the hardcopy forever. And we're going to want to share the PDFs with the wider research community at some point.

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 8]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+
+### Summary/Todo:
+
+Completed 7 Hours
+
+## Wednesday, Dec. 30
+
+### Relevant Notes:
+- **p.1410 added comment regarding values on section.
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:433]: is this supposed to be bolded?

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 9]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+    - ***Come back here***
+- p.1541.b added new entries
+- p.1551 fixed entry
+- p.1556 added av
+- p,1558 added to new entries
+
+**(mostly) Completed Geneva**
+
+
+### Beginning Smoot-Hawley
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:459]: Why is there nothing under this?

Also, an earlier entry said you were beginning Smoot Hawley. I'm confused.

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 10]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+
+### Misc
+
+- Page 149 missing in Geneva PDF?
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:463]: AAARGH

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 11]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+
+Complted 3.5 hours
+
+
+## Thursday, Dec. 31
+
+### Overview:
+
+- Begin Smoot-Hawley
+    - Most comments are included in "Extra" category for HS
```
Comment (kbuzard) [inline 2020_Winter_RA_Report_Gonzalez.md:475]: Does this refer to what you are doing, or what was already in the file?

File: 2020_Winter_RA_Report_Gonzalez.md [hunk 12]
```diff
@@ -139,16 +139,429 @@ Victor provided me with some helpful information related to some of my questions
...
+
+### Overview
+
+- Smoot-Hawley, schedule 8
+- p.904. Difficult to categorize tax rates. My issues are documented in the spreadsheet.
+- **POTENTIAL DIFFERENCE BETWEEN PDF AND HARDCOPY p.911.a** 
+- p.1108,1109,1119: difficult paragraphs
+
+
+Question/Thought: HS does not use tax intervals and does not specify rates based on value. As such, there are many instances with a single tax rate across entire paragraphs: **p.804 ass an example**
```
Comment (Zeyuan-Xiong) [inline 2020_Winter_RA_Report_Gonzalez.md]: This is also what I felt difficult about when coding the first two schedules, we probably should discuss about it next meeting. Also I notice that some missing paragraphs and subparagraphs that you made notes on in the EXTRA column are, or will be, completed after the entering of new lines from Geneva56. I will go through your notes in S-H again after finishing all the new lines.

**Review by kbuzard — COMMENTED** [review]
I've commented mostly on clarity of the notes.

I'm going to create a Github "issue" about the missing pages in Geneva so we remember to come back later and figure out what to do about this. Let's talk about this when we meet this week.

File: 2020_Winter_RA_Report_Gonzalez.md (hunk 3 above)
Comment (MatthewRGonzalez) [reply 2020_Winter_RA_Report_Gonzalez.md:319]: Based on your notes on this report, I have committed changes to this file. The new commit is available under "commits" and entitled "updating based on feedback from PR". I hope that this improves clarity. Thanks!

**Comment by kbuzard** [comment]
@MatthewRGonzalez Thanks for the updates! You're doing a great job with the documentation in general--thanks for your attention to this. The clarifications from the most recent commit are exactly what I was looking for!

Writing clear documentation is an art form (that I'm still learning), in particular because it's hard to remember to put in things that seem obvious to oneself but would not be to someone else. So I'll continue to read these at a high level and leave the nitty-gritty to you and @Zeyuan-Xiong to work out.

**Pull request merged**